core/src/com/unciv/models/ruleset/unique/Unique.kt

```diff
@@ -161,120 +159,129 @@ class Unique(val text: String, val sourceObjectType: UniqueTarget? = null, val s
             return 0
         }
 
-        return when (condition.type) {
-            // These are 'what to do' and not 'when to do' conditionals
-            UniqueType.ConditionalTimedUnique -> true
+        /** readability shortcuts */
+        fun param0() = condition.params[0]
```

Comment:
So I understand the attempt, and I understand we call params 0 and 1 constantly... But is this necessary? This is different from how we call params literally everywhere else

Reply:
We just use `unique.params[]`.

Reply:
E.g. `UniqueType.BuyUnitsIncreasingCost` uses more than 2 params, seen [here](https://github.com/yairm210/Unciv/blob/8d0853f0a951e33524f3ab5c0662f8b9b1da6431/core/src/com/unciv/logic/city/CityConstructions.kt#L672)